addons/main/XEH_preInit.sqf

```diff
@@ -2,7 +2,7 @@
 
 if (!isServer) exitWith {};
 
-private _category = "tfar_autoswitch";
+private _category = "Gruppe Adler TFAR Autoswitch";
```

Comment:
```suggestion
private _category = "GRAD TFAR Autoswitch";
```

I seem to remember us going with "GRAD" everywhere else

Reply:
Yes this is the case for most projects up until now, but - as already discussed in internal forum threads - I'm pushing to use the full "Gruppe Adler" wherever possible to improve brand recognition :) 
PRs for the other projects will follow _soonTM_, while updating to the new mod marketing stuff.

In this case it doesn't even make a difference, because this mod is only loaded on the Server, so no one will ever see the category name.

Reply:
GRAD > Gruppe Adler - its a tag after all. No one knows wtf is "Advanced Combat Environment", its ACE.

Reply:
I 100% agree if you're talking about variables, prefixes etc.
This - on the other hand - is a display name we're talking about and I mean we have plenty space in the category dropdown. So why should we only use the short version? 
No one knows what GRAD means. Gruppe Adler on the other hand is known (at least a bit). Besides that: We want the people to remember _"Gruppe Adler"_ and not only _"GRAD"_ or am I wrong?
The ACE comparison is flawed imho. Our "brand" goes much further than just a single Arma 3 Mod. We have multiple mods and other non-mod projects.

Reply:
Hier mal ein Screenshot der ultimativen _Brand Recognition_ :

![Screenshot_20211215_130009](https://user-images.githubusercontent.com/52833/146182925-86d63f1f-7cfe-43fe-a0ca-7355fda0704d.png)

Geiles Marketing und shice auf den User.

![mike drop](https://thumbs.gfycat.com/ImpracticalImpoliteGallowaycow-size_restricted.gif)

Reply:
Also, äh… im category dropdown ist tatsächlich genug Platz. Ich find Brand Recognition auch gut, aber wir müssen aufpassen daß das nicht zulasten der User geht.

Ein besseres analog als _ACE_ wäre _MBG_

Mondkalb hat unendlich viele Mods gebaut, ich find besser daß da überall nur "MBG" steht und nicht … was auch immer da stünde.
Ja, das könnt man jetz als Beleg sehen daß das Präfix für nicht-erkennbarkeit steht, aber wenn wir das (anders als Mondkalb) überall zu unserm Clan verlinken, reicht das.

Reply:
https://forum.gruppe-adler.de/topic/2793/marketing-eigenentwicklungen-mods-diskussion/ <-das ist der fragliche Thread?

Reply:
> Mondkalb hat unendlich viele Mods gebaut, ich find besser daß da überall nur "MBG" steht und nicht … was auch immer da stünde.

![](https://media4.giphy.com/media/NLxsMFuJeKCCWzhMue/giphy.gif?cid=6c09b952b692b59b076c85471cc5eb6a4c4dee3d3488e53a&rid=giphy.gif)

Reply:
3 Commando Brigade Factions
3 Commando Brigade Vehicles
Community Upgrade Project Terrains

...

das is bloat und bringt null